DOCSP-36218: builders with dataclass properties (#186)
mongodb/docs-kotlin: source/fundamentals/builders/aggregates.txt

.. literalinclude:: /examples/generated/AggregatesBuilderTest.snippet.movie-data-class.kt
   :language: kotlin

> .. tip:: Aggregates and Data Class Properties
> 
>    You can use methods from the ``Aggregates`` and
>    ``Accumulators`` classes directly with data class
>    properties by adding the optional {+driver-short+} extensions
>    dependency to your application. To learn more and view examples, see
>    the :ref:`kotlin-builders-data-classes` guide.

Reviewer: If you make these changes, you can use an include in all the Builders files instead of a c+p with minor revisions:
```suggestion
.. tip:: Builders and Data Class Properties

   You can use this builders methods and classes directly with data class
   properties by adding the optional {+driver-short+} extensions
   dependency to your application. To learn more and view examples, see
   the :ref:`kotlin-builders-data-classes` guide.
```

Author: good idea
